lib/error.js

```diff
@@ -233,6 +239,15 @@ AppError.unverifiedAccount = function () {
   })
 }
 
+AppError.unverifiedSession = function () {
```

Comment:
...he also thinks this should be inserted further down, so it's in `ERRNO`-order.